packages/repo-tools/src/generate-sponsors.ts

```diff
@@ -184,7 +184,7 @@ async function stringifyObject(
     2,
   );
 
-  return prettier.format(text, {
+  return await prettier.format(text, {
```

Comment:
q: why await?

Reply:
yes i get thet,but this function returns promise, and we don't have to await here

Reply:
Going off of https://typescript-eslint.io/rules/return-await - I'm intentionally adding it to get all of the function in any error stack.